Automattic/woocommerce-payments
# feat: BNPL April announcement

Fixes https://github.com/Automattic/woocommerce-payments/issues/8591

#### Changes proposed in this Pull Request









Adding a dialog to the admin area for merchants, to inform them that they can enable BNPL methods.

#### Testing instructions





- Ensure you don't have any BNPL methods enabled
- Ensure you haven't seen the dialog before (if you did, just use the filter below)
```php
		add_action( 'get_user_metadata', function ( $_value, $_object_id, $meta_key ) {
			if ( $meta_key !== '_wcpay_bnpl_april15_viewed' ) {
				return $_value;
			}

			return '0';
		}, 11, 3 );
```
- Ensure you have at least 3 successful orders (if you don't, create them)
- Ensure your account is set in one of the eligible countries (e.g.: `US`)
- Navigate to one of the WooPayment pages (besides the "Disputes" page)
  - PLEASE NOTE: this doesn't work for "soft refreshes" (e.g.: navigating from the main WC page to Payments > Overview - there needs to be a page refresh in between)
- You should see the dialog

-------------------


**Post merge**
